src/pages/meta-tag.js

```diff
@@ -1,16 +1,16 @@
 import React, { useState } from 'react'
-import { Container } from '@material-ui/core'
-import Checkbox from '@material-ui/core/Checkbox'
-import FormControlLabel from '@material-ui/core/FormControlLabel'
+import { Container } from '@mui/material'
+import Checkbox from '@mui/material/Checkbox'
```

Comment:
Leaving this alone because I will need to rebase on https://github.com/WICG/webmonetization/pull/319